integration/src/main/resources/suites/FullPerf.json

```diff
@@ -1,9 +1,11 @@
 {
   "name": "FullPerf",
-  "description": "All perf tests",
-  "serverSpecificationFile": "drshub-perf.json",
+  "description": "All perf tests locally in the absence of a live perf environment (see: DR-3380)",
```

Comment:
Filed https://broadworkbench.atlassian.net/browse/DR-3380 for complete clean-up of artifacts from now-defunct perf environment, and planning how we'd like to run these tests in the future.